### PR TITLE
Mission Control Container Scaffolding (#6) -MR

### PR DESCRIPTION
**Pull request by MxMatRam**
3RD/4TH COMMITS:
*declared MissionControlComponent in Mission Control module
*deleted Missions folder+contents
*Zelda resolved merging file conflicts

SECOND COMMIT:
*created Missions module with lazy-loading in app-routing-module

FIRST COMMIT:
*created Mission Control module and container files for app scaffolding
*attempted top-level routing in mission-control-routing file
*did NOT complete lazy-loading at the time of upload

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { BrowserModule } from '@angular/platform-browser';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { SharedModule } from './shared/shared.module';
+import { MissionControlComponent } from './app/mission-control/mission-control.component';
```
Comment (ZeldaMazzy) [inline src/app/app.module.ts]: remove this import

File: src/app/missions/missions-routing.module.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```
Comment (ZeldaMazzy) [inline src/app/missions/missions-routing.module.ts]: delet this file

File: src/app/missions/missions.module.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
```
Comment (ZeldaMazzy) [inline src/app/missions/missions.module.ts]: delete

**Review by ZeldaMazzy — CHANGES_REQUESTED** [review]
Declare the Mission Control component in the Mission Control module

File: src/app/app.module.ts (hunk 1 above)
Comment (MxMatRam) [reply src/app/app.module.ts]: deleted

File: src/app/missions/missions-routing.module.ts (hunk 2 above)
Comment (MxMatRam) [reply src/app/missions/missions-routing.module.ts]: deleted

File: src/app/missions/missions.module.ts (hunk 3 above)
Comment (MxMatRam) [reply src/app/missions/missions.module.ts]: deleted

**Comment by MxMatRam** [comment]
> Declare the Mission Control component in the Mission Control module

declared and deleted Missions folder+contents

**Review by ZeldaMazzy — APPROVED** [review]

**Pull request merged**